WordPress/src/main/java/org/wordpress/android/ui/main/WPMainActivity.java

```diff
@@ -288,6 +291,11 @@ public void onPageScrolled(int position, float positionOffset, int positionOffse
                             ShortcutsNavigator.ACTION_OPEN_SHORTCUT), this, getSelectedSite());
                 } else if (openRequestedTab) {
                     handleOpenTabIntent(getIntent());
+                } else if (openedFromBookmarkSnackbar) {
```

Comment:
Opening reader tab on a specific filter feels quite similar to opening a tab. We might consider reusing the openRequestedTab route with an extra intent parameter. WDYT?

Reply:
Good call!